pkg/cmd/cli/cmd/edit.go

```diff
@@ -417,7 +417,7 @@ func applyPatch(delta *jsonmerge.Delta, info *resource.Info, version string) err
 // their updates were preserved.
 func preservedFile(err error, path string, out io.Writer) error {
 	if len(path) > 0 {
-		if _, err := os.Stat(path); !os.IsNotExist(err) {
+		if _, err = os.Stat(path); !os.IsNotExist(err) {
```

Comment:
@liggitt It isn't as scary as it looks. The code originally over-wrote the value of `err` but made it look like it didn't with `:=`. Either we change to `=` and make it obvious that this is happening or change the error name to make it not happen. I'll take the latter path as it seems safer.